extensions/positron-notebook-controllers/src/notebookController.ts

```diff
@@ -182,81 +182,81 @@ export class NotebookController implements vscode.Disposable {
 		// Clear any existing outputs.
 		currentExecution.clearOutput();
 
-		// Create a promise that resolves when the cell execution is complete i.e. when the runtime
```

Comment:
These changes are mostly to make sure that messages are processed sequentially. We could use the `executeCode` function below as the basis for https://github.com/posit-dev/positron/issues/2622.